src/JsonSchema/Constraints/Constraint.php

```diff
@@ -61,12 +61,20 @@ public function setUriRetriever(UriRetriever $uriRetriever)
     /**
      * {@inheritDoc}
      */
-    public function addError($path, $message)
+    public function addError($path, $message, $constraint='', $more=null)
     {
-        $this->errors[] = array(
+        $a = array(
```

Comment:
Hmm, it's better to use meaningless very short names for variables that are being used temporarily and locally within such a small method, but fine I'll change the name.